config/object_store_conf.xml.sample

```diff
@@ -46,22 +46,44 @@
         <object_store type="azure_blob">
             <auth account_name="..." account_key="...." />
             <container name="unique_container_name" max_chunk_size="250"/>
-            <cache path="database/object_store_cache" size="100" />
+            <cache path="database/object_store_cache" size="100" /> <!-- size is in gigabytes -->
             <extra_dir type="job_work" path="database/job_working_directory_azure"/>
             <extra_dir type="temp" path="database/tmp_azure"/>
         </object_store>
         -->
 
-        <!-- Sample Cloud Object Store
-        <object_store type="cloud">
+        <!-- Cloud ObjectStore: Amazon Simple Storage Service (S3) -->
+        <!--
+        <object_store type="cloud" provider="aws" order="0">
             <auth access_key="..." secret_key="..." />
             <bucket name="..." use_reduced_redundancy="False" />
-            <cache path="database/object_store_cache" size="1000" />
+            <cache path="database/object_store_cache" size="100" /> <!-- size is in gigabytes -->
             <extra_dir type="job_work" path="database/job_working_directory_s3"/>
             <extra_dir type="temp" path="database/tmp_s3"/>
         </object_store>
         -->
 
+        <!-- Cloud ObjectStore: Microsoft Azure Blob Storage -->
+        <!--
+        <object_store type="cloud" provider="azure" order="0">
+            <auth subscription_id="..." client_id="..." secret="..." tenant="..." />
+            <bucket name="..." use_reduced_redundancy="False" />
+            <cache path="database/object_store_cache" size="100" /> <!-- size is in gigabytes -->
+            <extra_dir type="job_work" path="database/job_working_directory_azure"/>
+            <extra_dir type="temp" path="database/tmp_azure"/>
+        </object_store>
+        -->
+
+        <!-- Cloud ObjectStore: Google Compute Engine (GCE) -->
+        <!--
+        <object_store type="cloud" provider="google" order="0">
+            <auth credentials_file="..." />
+            <bucket name="..." use_reduced_redundancy="False" />
+            <cache path="database/object_store_cache" size="1000" /> <!-- size is in gigabytes -->
+            <extra_dir type="job_work" path="database/job_working_directory_gce"/>
+            <extra_dir type="temp" path="database/tmp_gce"/>
```

Comment:
Perhaps better to go with the Google Cloud **Platform** (GCP) terminology vs. GCE so it's consistent with CloudBridge (and Google). xref https://github.com/CloudVE/cloudbridge/pull/189

Reply:
Done.